CHANGELOG.md

```diff
@@ -18,7 +18,7 @@ All notable changes to this project will be documented in this file, in reverse
 
 ### Removed
 
-- Nothing.
+- [#223](https://github.com/zendframework/zend-mail/pull/223) removes support for zend-stdlib v2 releases.
```

Comment:
why?

Reply:
v2 is no longer supported: https://framework.zend.com/long-term-support
and it's not fully compatible with PHP 7.3. No point to keep it here.